setup/account_multi_vat_sale/.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
+
+This directory caches those eggs to prevent repeated downloads.
+
+However, it is safe to delete this directory.
+
```

Comment:
The .eggs directory should not be in get. I'd recommend you put it in your global gitignore (`git config --global core.excludesFile '~/.gitignore'`).